src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+(function() {
+    let translations = {
+            'Passwords'                           : 'Mots de passe',
+            'All'                                 : 'Tous',
+            'Folders'                             : 'Dossiers',
+            'Recent'                              : 'Récents',
```

Comment:
Just to consistent with NextCloud French traduction, the 's' should be removed => `Récent`